sdk/core/core-amqp/src/retry.ts

```diff
@@ -26,6 +26,27 @@ function isDelivery(obj: any): boolean {
   return result;
 }
 
+/**
+ * TODO: We should question whether it's even a legitimate way of setting the timeout
+ * because it just squashes all timeouts beneath 60 seconds to be 60 seconds instead.
+ *
+ * @export
+ * @param {(RetryOptions | undefined)} retryOptions
+ * @returns {number}
+ */
+export function getRetryAttemptTimeoutInMs(retryOptions: RetryOptions | undefined): number {
```

Comment:
Are we just going to remove this function anyways though when you go through and obliterate the "must be 60 seconds or less"? Ie, should we just not bother doing this so we don't have to re-release core-amqp?